pkg/api/pod/util_test.go

```diff
@@ -1935,7 +1935,7 @@ func TestDropSubPathExpr(t *testing.T) {
 		},
 	}
 
-	for _, enabled := range []bool{true, false} {
+	for _, enabled := range []bool{true} {
```

Comment:
drop the loop if it's not serving any purpose

Reply:
Dropped